.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+### Changes
+
+Please describe both what is changing and why this is important. Include:
+
+- Endpoints added, deleted, deprecated, or changed
+- Classes and methods added, deleted, deprecated, or changed
+- Screenshots of new or changed UI, if applicable
+- A summary of usage if this is a new feature or change to a public API (this should also be added to relevant documentation once released)
+- Any alternative designs or approaches considered
+
+### References
+
+Please include relevant links supporting this change such as a:
+
+- support ticket
+- community post
+- StackOverflow post
+- support forum thread
+
+### Testing
```

Comment:
I'm not seeing any tests in this repo, unit or otherwise.

Reply:
Remove the whole section?  Might discourage people adding tests.

Reply:
I would remove anything that makes it seem like there are tests to add to. "Adding test coverage" for a library that doesn't have tests already is basically "adding a test suite," which we should probably do ourselves.